frontend/public/images/Frame 2.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="360" height="800" viewBox="0 0 360 800" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
možná jsi to udělala, ale svg soubory je dobrý prohnat timhle nástrojem https://jakearchibald.github.io/svgomg/

protože třeba ty 2 groupy <g id="..."> tu určitě nejsou potřeba, neni potřeba ani to id u <path>.

A nepojmenovával bych soubor s mezerou v názvu